docs/modules/ROOT/pages/index.adoc

```diff
@@ -503,21 +513,188 @@ In each of the above sections, the following changes MUST be applied:
+
+In this scenario, the _<<_identity_assertion_consumer,identity assertion consumer>>_ SHOULD make its trust decision based on the following predicates:
+
+. Does the _<<_identity_assertion_consumer,identity assertion consumer>>_ trust the *<<_identity_assertion,identity assertion>> generator* to request a credential summary from the _<<_credential_holder,credential holder>>_ and accurately reflect that credential summary into the *<<_identity_assertion,identity assertion>>?*
+.. Is there a direct trust relationship with the *<<_identity_assertion,identity assertion>> generator?* If so, proceed to step 2.
+.. Is there a transitive trust relationship with the *<<_identity_assertion,identity assertion>> generator* via its _credential issuer?_ (In other words, does the _<<_identity_assertion_consumer,identity assertion consumer>>_ trust the *<<_identity_assertion,identity assertion>> generator’s* _credential issuer_ to issue valid signature credentials?)
+.. If so, has the _credential issuer_ issued a revocation for the *<<_identity_assertion,identity assertion>> generator’s* credential? If so, do not proceed. The *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
+.. If the transitive trust relationship exists and the credential has not been revoked, proceed to step 2.
+.. If neither relationship can be demonstrated, do not proceed. The *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
+. Does the _<<_identity_assertion_consumer,identity assertion consumer>>_ trust the _<<_named_actor,named actor’s>> credential issuer_ to issue valid credentials?
+.. Is there a direct trust relationship with the _<<_named_actor,named actor’s>> credential issuer?_ If so, proceed to step 3.
+.. Is there a transitive trust relationship with the _<<_named_actor,named actor’s>> credential issuer_ via its _credential issuer?_ (In other words, does the _<<_identity_assertion_consumer,identity assertion consumer>>_ trust the _<<named_actor,named actor’s>> credential issuer_ to issue valid credentials?) If so, proceed to step 3.
+.. If neither relationship can be demonstrated, do not proceed. The *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
+. Has the _credential issuer_ issued a revocation for the _<<_named_actor,named actor’s>>_ credential?
+.. If so, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
+.. If no such revocation has been issued, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *trusted.*
+
+=== Threats to trust model
+
+_This section is non-normative._
+
+This section enumerates a number of potential attacks on the *<<_identity_assertion,identity assertion>>* trust model. If concrete guidance to mitigate or prevent a specific attack is available, that guidance should be incorporated as specific normative requirements elsewhere in this specification and referenced here.
+
+==== Replay attacks
+
```

Comment:
Proposed more official text:

Overview:
One class of attacks against an identity assertion would be any attempt to extract a valid identity assertion out of one asset and embed it in an asset of an attacker's choosing without causing a validation error. This scenario could allow an attacker to falsely attribute an asset to a victim's identity without their consent or knowledge. If the attacker's asset is controversial or illegal, then falsely attributing it to a victim's identity could result in severe consequences for the victim.

Reply:
@puhley good suggestion. I've adopted (and slightly adapted) this language.